### ISSUE TITLE
People API issue with multiple libraries in Podfile

### ISSUE THREAD

**Issue by kyrylyermak** [google/google-api-objectivec-client-for-rest#637]
Having an issue with this two pods:
```
pod 'GoogleAPIClientForREST/Drive'
pod 'GoogleAPIClientForREST/People'
```

If I'm trying to get them together, cocoa pods installs version 1.2.1 to both of them, when I specify version, and changed to:

```
pod 'GoogleAPIClientForREST/Drive', '3.5.1'
pod 'GoogleAPIClientForREST/People', '3.5.1'
```
  
I will get this type of error:

> [!] CocoaPods could not find compatible versions for pod "GoogleAPIClientForREST/People":
>   In Podfile:
>     GoogleAPIClientForREST/People (= 3.5.1)
> 
> None of your spec sources contain a spec satisfying the dependency: `GoogleAPIClientForREST/People (= 3.5.1)`.
> 
> You have either:
>  * mistyped the name or version.
>  * not added the source repo that hosts the Podspec to your Podfile.
 
But version 1.2.1 is so old for today, and for example PeopleConnectionsList is not working at all.
How can I get latest version of this pods?

**Comment by thomasvl** [comment]
I don't think this is going to be specific to this project and likely more about your use of CocoaPods.  Do you maybe have a `Podfile.lock` locally/checked in?  You might need to consult the CocoaPods docs and make sure you do the proper steps for updating your dependencies to newer versions.

**Comment by kyrylyermak** [comment]
Nothing special in this case for `Podfile.lock` I think

Can't be real, because I can update `Drive` to the latest version, but not for `People` library

**Comment by thomasvl** [comment]
Wait, did People go away or get renamed?  The subspecs we offer is based on what are listed as active services. So you may have some other migration to do.

**Comment by kyrylyermak** [comment]
No, People didn't go away or get renamed. 
The problem is that I can't install the latest version of the `People` library, your lates release is 3.5.1, but only one version is available 1.2.1

P.S. for `Drive` subspec everything is ok, but not for `People`, same steps for updating the pod, everything is same, even tried to fully delete your pods from project, and install again - nothing works

**Comment by thomasvl** [comment]
Then you might need to ask on some CocoaPods support spaces, it seems like it should be able to handle the removal/addition of a subspec dependency, not sure why it isn't in this case.
